spec/runtime/loader.spec.ts

```diff
@@ -320,7 +322,7 @@ describe('loadStack', () => {
             {
               name: 'FOO',
               type: 'string',
-              input: { type: 'text', validationRegex: 'w+' },
+              textInput: { validationRegex: 'w+' },
```

Comment:
also... and please don't kill me here, it didn't go from `{input: { type: 'text', other: field }}` to `{textInput: {other: field}}` it went to `{input: { text: {other: field }}}`. In proto, input will be a message that has a oneof for the input types.